corehq/apps/api/resources/v0_5.py

```diff
@@ -1179,6 +1181,7 @@ def cursor_query(cls, domain: str, local_timezone: pytz.tzinfo.DstTzInfo, params
 
         queryset = queryset.annotate(UTC_start_time=Min('event_date'), UTC_end_time=Max('event_date'))
 
+        cls.count = queryset.count()
```

Comment:
Shouldn't this be inside the `override_settings` block?  That is, won't the date bucketing affect the number of results returned?

Reply:
Yes you're right, [cf0abd1](https://github.com/dimagi/commcare-hq/pull/32922/commits/cf0abd16d98b7641c07156fa6392be87c61abe8d)

Also added a test that would have caught this